pkg/operator/ceph/file/mds/spec.go

```diff
@@ -44,8 +45,8 @@ const (
 func (c *Cluster) makeDeployment(mdsConfig *mdsConfig, namespace string) (*apps.Deployment, error) {
 
 	mdsContainer := c.makeMdsDaemonContainer(mdsConfig)
-	mdsContainer = config.ConfigureStartupProbe(cephv1.KeyMds, mdsContainer, c.clusterSpec.HealthCheck)
-	mdsContainer = config.ConfigureLivenessProbe(cephv1.KeyMds, mdsContainer, c.clusterSpec.HealthCheck)
+	configureStartupProbe(&mdsContainer, c.fs.Spec.StatusCheck)
+	configureLivenessProbe(&mdsContainer, c.fs.Spec.StatusCheck)
```

Comment:
Why are we removing deduplication here by not using `config.ConfigureLivenessProbe`? See how the `CephObjectStore` spec handles this if you need to see usage.

Reply:
Even `CephObjectStore` works this way only, 

By travis suggestion of using like this `func ConfigureStartupProbe(daemon cephv1.KeyType, container *v1.Container, livenessProbe <type>) `
This will also be updated, for using `config.ConfigureLivenessProbe`